source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-10-06](#release-2022-10-06)
 - [Release 2022-09-15](#release-2022-09-15)
 - [Release 2022-09-08](#release-2022-09-08)
 - [Release 2022-09-01](#release-2022-09-01)
 - [Release 2022-08-25](#release-2022-08-25)
 - [Release 2022-08-18](#release-2022-08-18)
-- [Release 2022-08-10](#release-2022-08-10)
+
+## Release 2022-10-06
+
+### Improvements
+
+#### User Interface (UI)
+ - Added Insights to the channel switcher.
+ - Added a button to easily copy the content of text or code files in file previews.
+ - The team unread icon for muted channels is now hidden in the sidebar.
+ - Updated the designs for the “payment failed” email notifications.
+
+#### Administration
+ - A ``batchSize`` option has been added to the ``mattermost export`` CLI command to limit the number of items exported. By default, if it is not included, it exports all posts.
```

Comment:
do we mean `mmctl` export?

Reply:
If we are referring to the CLI command, docs updates for v7.4 will be available here: https://docs.mattermost.com/manage/command-line-tools.html#mattermost-export

Reply:
This is referring to the CLI command.